service/history/workflow/mutable_state_impl.go

```diff
@@ -4260,10 +4257,9 @@ func (ms *MutableStateImpl) eventsToReplicationTask(
 }
 
 func (ms *MutableStateImpl) syncActivityToReplicationTask(
-	now time.Time,
 	transactionPolicy TransactionPolicy,
 ) []tasks.Task {
-
+	now := time.Now().UTC()
```

Comment:
for active cluster, passing a now which is time.Now() vs create the now locally makes no difference
especially this now is used for metrics calculation

Reply:
passive cluster will not generate sync activity task